<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Impostazioni</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Stanza</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="1" column="0">
            <widget class="QLabel" name="widthLabel">
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="midLineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>Larghezza Stanza</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="widthEdit">
             <property name="suffix">
              <string> m</string>
             </property>
             <property name="minimum">
              <double>1.000000000000000</double>
             </property>
             <property name="maximum">
              <double>99.989999999999995</double>
             </property>
             <property name="value">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="heightLabel">
             <property name="text">
              <string>Lunghezza Stanza</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="portServerLabel">
             <property name="text">
              <string>Porta Server</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="portServerEdit">
             <property name="minimum">
              <number>1024</number>
             </property>
             <property name="maximum">
              <number>65535</number>
             </property>
             <property name="value">
              <number>27015</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="boardLabel">
             <property name="text">
              <string>Schedine</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QTableWidget" name="boardTable"/>
           </item>
           <item row="5" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="addBoard">
               <property name="text">
                <string>Aggiungi</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="modBoard">
               <property name="text">
                <string>Modifica</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeBoard">
               <property name="text">
                <string>Rimuovi</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="heightEdit">
             <property name="suffix">
              <string> m</string>
             </property>
             <property name="minimum">
              <double>1.000000000000000</double>
             </property>
             <property name="maximum">
              <double>99.989999999999995</double>
             </property>
             <property name="value">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Monitoraggio</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="nLabel">
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="text">
            <string>Costante di propagazione del segnale (n)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="minLabel">
           <property name="text">
            <string>Minimo minuti per conteggio presenza</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="minEdit">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="retryLabel">
           <property name="text">
            <string>Limite fallimenti aggregazione</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="retryEdit">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="secretLabel">
           <property name="text">
            <string>Segreto Condiviso</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="secretEdit">
             <property name="text">
              <string>segreto</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="secretEcho">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../res.qrc">
               <normaloff>:/resources/resources/see_pass.svg</normaloff>:/resources/resources/see_pass.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="nEdit">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Database</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Per il corretto funzionamento del programma ti consigliamo di preparare il database avendo cura di inserire correttamente i dati sottostanti.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="1" column="0">
          <widget class="QLabel" name="hostLabel">
           <property name="text">
            <string>Host</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="hostEdit">
           <property name="text">
            <string>localhost</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="dbLabel">
           <property name="text">
            <string>Nome Database</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="dbEdit">
           <property name="text">
            <string>data</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="portLabel">
           <property name="text">
            <string>Porta</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="userLabel">
           <property name="text">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="userEdit">
           <property name="text">
            <string>root</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="passLabel">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="passEdit">
             <property name="text">
              <string>NewRoot12Kz</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="passEcho">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../res.qrc">
               <normaloff>:/resources/resources/see_pass.svg</normaloff>:/resources/resources/see_pass.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="tableLabel">
           <property name="text">
            <string>Nome Tabella</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="tableEdit">
             <property name="text">
              <string>stanza</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="restoreBoardButton">
             <property name="text">
              <string>Ripristina Schede</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="portEdit">
           <property name="minimum">
            <number>1024</number>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>3306</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="resetCheck">
         <property name="text">
          <string>Inizializza Database quando applico le impostazioni</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>MAC Nascosti</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&lt;p&gt;Configurazione parametri e tolleranze per Analisi MAC nascosti.&lt;/p&gt;

&lt;ul&gt;
&lt;li&gt;&lt;b&gt;Tolleranza&lt;/b&gt;: Valore della differenza per cui la percentuale di somiglianza è pari a 0. &lt;/li&gt;
&lt;li&gt;&lt;b&gt;Peso&lt;/b&gt;: Numero intero che rappresenta il peso che viene dato ad ogni parametro.&lt;/li&gt;
&lt;/ul&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&lt;b&gt;Posizione&lt;/b&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2">
           <item>
            <widget class="QDoubleSpinBox" name="posTolEdit">
             <property name="suffix">
              <string> m</string>
             </property>
             <property name="minimum">
              <double>0.000000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="posPesoEdit">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&lt;b&gt;Tempo&lt;/b&gt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
           <item>
            <widget class="QSpinBox" name="timeTolEdit">
             <property name="suffix">
              <string> s</string>
             </property>
             <property name="maximum">
              <number>86400</number>
             </property>
             <property name="value">
              <number>3600</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="timePesoEdit">
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>&lt;b&gt;SSID&lt;/b&gt;</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QSpinBox" name="ssidPesoEdit"/>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,2">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>&lt;b&gt;Tolleranza&lt;/b</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>&lt;b&gt;Peso&lt;/b&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::RestoreDefaults</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>widthEdit</tabstop>
  <tabstop>heightEdit</tabstop>
  <tabstop>portServerEdit</tabstop>
  <tabstop>boardTable</tabstop>
  <tabstop>addBoard</tabstop>
  <tabstop>modBoard</tabstop>
  <tabstop>removeBoard</tabstop>
  <tabstop>nEdit</tabstop>
  <tabstop>minEdit</tabstop>
  <tabstop>retryEdit</tabstop>
  <tabstop>secretEdit</tabstop>
  <tabstop>secretEcho</tabstop>
  <tabstop>hostEdit</tabstop>
  <tabstop>dbEdit</tabstop>
  <tabstop>portEdit</tabstop>
  <tabstop>userEdit</tabstop>
  <tabstop>passEdit</tabstop>
  <tabstop>passEcho</tabstop>
  <tabstop>tableEdit</tabstop>
  <tabstop>restoreBoardButton</tabstop>
  <tabstop>resetCheck</tabstop>
  <tabstop>posTolEdit</tabstop>
  <tabstop>posPesoEdit</tabstop>
  <tabstop>timeTolEdit</tabstop>
  <tabstop>timePesoEdit</tabstop>
  <tabstop>ssidPesoEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
